cmd/podman/images/scp.go

```diff
@@ -0,0 +1,410 @@
+package images
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net"
+	urlP "net/url"
+	"os"
+	"os/user"
+	"strings"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/parse"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/domain/infra"
+	t "github.com/containers/podman/v3/pkg/terminal"
+	"github.com/containers/podman/v3/pkg/util"
+	scpD "github.com/dtylman/scp"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
+)
+
+var (
+	validScpFormats = []string{define.OCIManifestDir, define.OCIArchive, define.V2s2ManifestDir, define.V2s2Archive}
+)
+
+var (
+	saveScpDescription = `Securely copy an image from one host to another.`
+
+	imageScpCommand = &cobra.Command{
+		Use:   "scp [options] IMAGE [IMAGE...]",
+		Long:  saveScpDescription,
+		Short: "securely copy images",
+		RunE:  scp,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
```

Comment:
Does anything prevent `len(args) > 2`?

Reply:
@cdoern 
use
Args:              cobra.RangeArgs(1, 2),

Reply:
Still outstanding I think.

Reply:
changed to MinNumArgs, has same effect I think? RangeArgs(1,2) throws strange errors with images